.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+Thank you for opening an issue!
+
+Issues are meant to serve two purposes: **reporting bugs**, or **suggesting features**. Please use whichever template matches your issue.
+
+For all other matters, like support for using Guppy, the easiest way to chat with us is through Gitter: https://gitter.im/guppy-gui/Lobby
+
+---
+
+### Reporting Bugs
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
```

Comment:
Examples should be for desktops I think.

Reply:
Hah, good catch. I'm surprised this was the default suggested by Github!